pkg/ovs/util.go

```diff
@@ -112,6 +112,9 @@ func parseDHCPOptions(raw string) map[string]string {
 		if len(kv) != 2 || len(kv[0]) == 0 || len(kv[1]) == 0 {
 			continue
 		}
+		if strings.HasPrefix(kv[0], "dns_server=") {
```

Comment:
```suggestion
		if kv[0] == "dns_server=" {
```

Reply:
if kv[0] == "dns_server="  should be  if kv[0] == "dns_server" ?

Reply:
> if kv[0] == "dns_server=" should be if kv[0] == "dns_server" ?

Yes.